TagSDK/Exceptions/TagSDKException.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Newtonsoft.Json;
+using RestSharp;
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Text;
+
+namespace TagSDK.Exceptions
+{
+    [Serializable]
+    public class TagSDKException : Exception
+    {
+        public HttpStatusCode StatusCode { get; set; }
+        public ResponseError Error { get; set; }
+
+        public TagSDKException() { }
+
+        public TagSDKException(string message)
+            : base(message) { }
+
+        public TagSDKException(HttpStatusCode StatusCode)
```

Comment:
```suggestion
        public TagSDKException(HttpStatusCode statusCode)
```


Parameters must be camelCase